helpdesk_mgmt/README.rst

```diff
@@ -23,7 +23,7 @@ Helpdesk Management
     :target: https://runbot.odoo-community.org/runbot/282/11.0
     :alt: Try me on Runbot
 
-|badge1| |badge2| |badge3| |badge4| |badge5| 
+|badge1| |badge2| |badge3| |badge4| |badge5|
```

Comment:
You are not generating the README through the scripts and touching manually, and thus this is being changed. Please restore this line as it was, use the tool for generating it, or let the bot to do it on merge.

Reply:
I think it should be ok now, correct me if I'm wrong please.